### PR TITLE
Mention old node version in readme

### PR DESCRIPTION
**Pull request by emersonthis**
## Overview

* Adds a note about the older node version requirement
* Adds a .nvmrc file specifying v11

### CODE REVIEW

**Comment by calebeby** [comment]
Can we use node 10 instead? Node 11 is already EOL, and node 10 is maintenance LTS until april 2021

https://github.com/nodejs/release#release-schedule

**Comment by emersonthis** [comment]
@calebeby fine by me... I just used the newest version that allowed me to get the site running... Do you already know that v10 works? Or is that something I'd need to test. 

(PS: we shouldn't agonize over this too much because we're about to rebuild the whole thing in Eleventy)

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+10
```
Comment (tylersticka) [inline .nvmrc:1]: - No newline at end of file

**Comment by emersonthis** [comment]
I updated node in PR #206 so this is no longer relevant

**Pull request closed without merging**